src/unit_tests/wazuh_db/test_wdb_agents.c

```diff
@@ -468,6 +468,17 @@ void test_wdb_agents_insert_vuln_cves_success_pkg_found(void **state)
 
 /* Tests wdb_agents_update_status_vuln_cves*/
 
+void test_wdb_agents_update_status_vuln_cves_statement_parameter_fail(void **state){
```

Comment:
Could you leave a space before opening curly brackets?

Reply:
Done! Thank you.